### DISCUSSION TITLE
Is it a shadcn/ui-like?

### DISCUSSION THREAD

**Discussion by pochka15** [themesberg/flowbite-react#1203] [General]
I'm just curious if flowbite-react is going to be a [chadcn/ui](https://ui.shadcn.com/) competitor? What makes flowbite-react unique?

**Comment by rluders** [comment]
Both are component libraries. I'm not sure if I would call it a competitor, both libraries have its unique styles, concepts, and approaches to solve similar problems. Flowbite-react implements the flowbite.com components to react world.

**Comment by SutuSebastian** [comment]
We will offer in the near future the DX and features of shadcn's CLI.

So far we've got the [`npx flowbite-react create`](https://flowbite-react.com/docs/getting-started/cli#flowbite-react-create) command, but here's a sneak peak over the roadmap:

CLI ideas:

- [ ] npx flowbite-react add [component-name]
    - [ ] locally fork components using https://jsrepo.dev/
    - [ ] JS/TS
- [ ] npx flowbite-react doctor (maybe run the `init` command with `—dry` param)
    - [ ] checks if config files exist
    - [ ] checks if tailwind source/content paths are correct
- [ ] npx flowbite-react update/upgrade
    - [ ] handle the package update and migration (using codemod)
- [ ] npx flowbite-react prefix
    - [ ] either detect the prefix or prompt for it
    - [ ] setup the prefix in the config file + render `ThemeConfig` component with the prefix value


TODO:

- [ ] Monorepo support
- [ ] repair `.flowbite-react/config.json` config file on save/read
- [ ] auto repair relative `class-list.json` path in (options) when files are moved
    - [ ] tailwind.config.js
    - [ ] *.css file
